bc_obps/registration/migrations/0087_remove_duplicate_contacts.py

```diff
@@ -0,0 +1,69 @@
+# Generated by Django 5.0.13 on 2025-03-11 22:09
+
+from typing import Dict
+from django.db.models import Count
+from django.db import migrations
+from collections import defaultdict
+
+"""
+One-time forward-only migration to be applied to prod data.
+Purpose: First, check that contacts records are duplicated in full. Then,remove duplicate contacts.  Where operation.point_of_contact was one of the duplicates, replace with the unique contact.
+"""
+
+
+def count_stats(Contact) -> Dict[str, int]:
+    """Collects and returns key statistics about the data before and after migration."""
+    return {
+        'total_contacts': Contact.objects.count(),
+        'total_unique_contacts': Contact.objects.values(
+            'first_name', 'last_name', 'email', 'position_title', 'phone_number', 'business_role', 'operator', 'address'
+        )
+        .distinct()
+        .count(),
+        'total_unique_emails_and_operators': Contact.objects.values('email', 'operator').distinct().count(),
```

Comment:
This line is new since practice migrations. Once we created contacts from existing users, we ended up with some contacts that have the same email but different operators. This is because sometimes users would create an operator when theirs already existed, or make a new one instead of using the one imported from SWRS, etc. So we can only check for uniqueness of the email/operator combination, not email alone